tests/e2e/e2e_common.go

```diff
@@ -244,4 +249,10 @@ func setClusterFlavor(clusterFlavor cnstypes.CnsClusterFlavor) {
 	if strings.TrimSpace(string(kind)) == "RWX" {
 		rwxAccessMode = true
 	}
+
+	//Check if its windows env
+	workerNode := os.Getenv("WORKER_NODE")
+	if strings.TrimSpace(string(workerNode)) == "WINDOWS" {
```

Comment:
Need to make appropriate changes to the tests/e2e/docs/vanilla_cluster_setup.md file as well

Reply:
done